lib/connection.ts

```diff
@@ -349,57 +349,63 @@ export class Connection {
   }
 
   private _handleMessage = (event: MessageEvent) => {
-    const message: WebSocketResponse = JSON.parse(event.data);
+    let message_group: WebSocketResponse | [WebSocketResponse] = JSON.parse(event.data);
```

Comment:
Current type is list with 1 response in it. This indicates an array:

```suggestion
    let message_group: WebSocketResponse | WebSocketResponse[] = JSON.parse(event.data);
```